src/layers/ImageLayer.js

```diff
@@ -118,30 +143,42 @@ export default class ImageLayer extends CompositeLayer {
       modelMatrix
     } = this.props;
     const { dtype } = loader;
-    const { data, width, height, unprojectLensBounds } = this.state;
+    const { width, height, unprojectLensBounds } = this.state;
     if (!(width && height)) return null;
+    const bounds = [0, height, width, 0];
+    const sharedLayerProps = {
+      bounds,
+      id: `image-sub-layer-${bounds}-${id}-${z}`,
+      onHover,
+      pickable,
+      onClick,
+      modelMatrix,
+      opacity,
```

Comment:
Again, does this do anything?

Reply:
If this is regarding the previous comment, yes, all deck.gl layers take `opacity.`